CMakeLists.txt

```diff
@@ -1989,6 +1989,12 @@ if(HPX_WITH_VIM_YCM)
   hpx_info("VIM YouCompleteMe: run 'make configure_ycm' to copy config file to source directory and enable support in YCM. To enable automatic loading of configure file, add to your .vimrc option: \"let g:ycm_extra_conf_globlist = ['${CMAKE_SOURCE_DIR}/*']\"")
 endif()
 
+################################################################################
+# Add rpm packaging
+if(HPX_WITH_RPM)
```

Comment:
Why is not enabled by default?

Reply:
I was thinking that this is a special use case and if one will build the package, one should explicitly enable it. I am open to enable it as default too.

Reply:
Since it is just a target which has to be initiated explicitly, I don't see a problem having it optionally. If it however requires `rpm` or anything else to be installed, it needs to stay optional. Ideally however, it would be even nicer, if it could be implemented completely outside of our CMakeLists.txt infrastructure. I have no idea what that all entails. There used to be OpenSUSE packages here: https://build.opensuse.org/package/show/Education/hpx

No idea what happened there.

Reply:
Yes, one would need to install packages to build the rpm or debian package. Therefore, it should be optional. Let me get into contact with the maintainer for the opensuse packages.

Reply:
I made a quick update of the OpenSUSE package: https://build.opensuse.org/package/show/home:cjunghans:branches:Education/hpx

Reply:
Here is v1.1.0 https://build.opensuse.org/package/show/science:HPC/hpx